### DISCUSSION TITLE
Compatibility with Elvox 900

### DISCUSSION THREAD

**Discussion by PilaScat** [PricelessToolkit/ESPBell-LITE#20] [Q&A] [answered]
HI, I wanted to buy and install the ESPBell-LITE but I don't have the necessary preparation to wire it.
Could you tell me if is it compatible and how to wire it?
[Datasheet](https://www.intercomsrus.com/handset_documentation/Elvox%20900%20Intercom%20Handset%20Data%20Sheet.pdf)
My doorbell:
![image](https://github.com/user-attachments/assets/1b2f05a6-fb73-41d5-84b8-53fb365749af)
Thanks

**Comment by PricelessToolkit** [comment]
Hi, yes it is compatible, you need to connect 
| ESPBell  | Intercom   |
|----------|------------|
| 1x Bell  | Terminal 3 |
| 1x Bell  | Terminal 6 |
| 2x Lock  | Terminal 3 |
| 2x Lock  | Terminal 7 |


| ESPBell  |Power Supply   |
|----------|-----------------|
|   GND    |       GND          |
|   VCC     |       5V              |

The lock has 4 wires 2 pairs

**Reply by PilaScat** [reply]
Like here then?
![intercom_wiring](https://github.com/user-attachments/assets/cc26c32a-6329-46a0-b6de-a15e29cc8d34)

**Comment by PilaScat** [comment]
I need to add a power supply?

**Reply by PricelessToolkit** [reply]
Yes, the LITE version requires a 5V power supply, as mentioned in the product description. It has the advantage of allowing you to open your door whenever you need. The MAX version, on the other hand, operates on a battery and only activates when the doorbell is pressed.

**Reply by PilaScat** [reply]
Do you have any suggestions? Something small that can fit in and maybe use the doorbell power.

**Reply by PilaScat** [reply]
![20240726_223508](https://github.com/user-attachments/assets/b8b6462b-a9f7-41c2-85d2-17718e836e4d)
This is my power supply

**Reply by PricelessToolkit** [reply]
This is a regular transformer, not a DC power supply. In the [Links section](https://github.com/PricelessToolkit/ESPBell-LITE?tab=readme-ov-file#links), you can find a link to a small "220V to 5V" power supply. However, working with mains voltage can be dangerous, so please avoid attempting this if you don't have experience. The safest option is to use a phone charger and a USB connector, like the one found [here](https://s.click.aliexpress.com/e/_DDCS7y3). Alternatively, you can cut a USB cable and connect the wires directly to the ESPBell wires.

**Reply by PilaScat** [reply]
Obviously I would do it with the current cut off, seeing it in this way is very simple, thank you.

**Comment by PricelessToolkit** [comment]
i found AC to DC converter https://s.click.aliexpress.com/e/_DBcgGvt 5-20V AC to 5-32V DC. you can try to use this to step down your doorbell transformer voltage, but for setting 5v you need a multimeter. but I can't say if it will work or not!

**Reply by PilaScat** [reply]
Seems better using the regular 220v ac to 5v dc, right?

**Reply by PricelessToolkit** [reply]
Yes

**Comment by PilaScat** [comment]
I tried to install it, but it doesn't turn on, could you tell me if I'm doing something wrong? I tried different USB cables and different power supplies without success, the multimeter says that little or nothing passes through the type c connector, could it be its fault?
![20240814_111255](https://github.com/user-attachments/assets/b6d9020f-3d01-4f06-80d8-ee812efa087b)
![20240814_111259](https://github.com/user-attachments/assets/8c54e9ee-d91d-45ae-baf3-ba9c877a791e)

**Reply by PricelessToolkit** [reply]
You need to use a basic 5V USB charger that consistently provides a 5V output, rather than a 'Power Delivery' or 'Quick Charge' charger. Newer chargers require information about the connected device to adjust the voltage output, which they achieve using additional wires in the cable and 5.1K resistors in the device. In this case, that setup is not possible. I believe you're using a OnePlus charger, which is designed for power delivery.

**Reply by PilaScat** [reply]
I actually tried these two, the oneplus rapid one was just close (the one in the picture is not pd or anything)
![20240814_132627](https://github.com/user-attachments/assets/e7d0b6d6-2c8a-4805-b0ab-0bbb8fb1a4b4)

**Comment by PricelessToolkit** [comment]
hmm, magic!, can you check if the USB connector is soldered correctly? zoom with the phone. or measure voltage with other "USB PCB"

**Reply by PilaScat** [reply]
I have only one connector, so I don't know how can I test more
![20240814_140104](https://github.com/user-attachments/assets/d7897e03-2eb7-4632-a470-1a631cab1e5b)
![20240814_140044](https://github.com/user-attachments/assets/2d9019cc-0725-4ac0-b01e-f4f34907cd3e)

**Comment by PricelessToolkit** [comment]
Do you also measure 0V when it's not connected to the ESP-Bell?

**Reply by PilaScat** [reply]
Yes, 0 or between 0 and 0.1, tried now

**Reply by PilaScat** [reply]
https://it.aliexpress.com/item/1005007148475800.html
bought this, seems cleaner

**Comment by PilaScat** [comment]
![20240826_150001](https://github.com/user-attachments/assets/1d458073-0b2c-41d5-b51d-858fb513e8df)
![20240826_145954](https://github.com/user-attachments/assets/dd25207c-0cd1-4e67-8bc1-4f70b8420dff)
Connected but only doorbeel is working, no lock, maybe I didnt understand the wiring for lock

**Reply by PilaScat** [reply]
Tried to connect to the right 7 but it open indefinitely

**Comment by PilaScat** [comment]
![20240826_152023](https://github.com/user-attachments/assets/fae9aa57-00bb-4f26-9d5a-4950dae8ad63)
In my hand there is 7, when i connect to 7 on the doorbell (gray wire next to it) it open the gate indefinitely, it started when I opened the gate from home assistant

**Reply by PricelessToolkit** [reply]
on, the Physical button is on 7 yes? and the button has 2 terminals. So, connect like in the photo, A to B or A to A, does not matter.
![gith](https://github.com/user-attachments/assets/c0b02ce0-f0ab-42e6-ba22-3bece93b91c8)

**Reply by PilaScat** [reply]
ahhhhhhhh now I understand, yes the button is open, I'll try now

**Reply by PilaScat** [reply]
![1724684717816246364567606267860](https://github.com/user-attachments/assets/6df2c0eb-3d5f-47b8-89d9-c3d0abd8c2aa)
Connected but even without power in the espbell it opens the gate indefinitely

**Reply by PilaScat** [reply]
Even inverted but same

**Reply by PricelessToolkit** [reply]
Without power, the relay is in the open position so no current can pass if the relay is not damaged, which is impossible it can handle 2.5A at 25v. 1. Can you try to open the gate one more time from HA? while espbell is connected.  2. disconnect BELL wires and check if it still keeps opening. 3. disconnect the power wire and check if it still keeps opening.

**Reply by PilaScat** [reply]
When espbell is not connected the gate remains closed, now espbell is disconnected because if I keep it connected it opens the gate forever, now I'll test all

**Reply by PilaScat** [reply]
> Without power, the relay is in the open position so no current can pass if the relay is not damaged, which is impossible it can handle 2.5A at 25v. 1. Can you try to open the gate one more time from HA? while espbell is connected. 2. disconnect BELL wires and check if it still keeps opening. 3. disconnect the power wire and check if it still keeps opening.

1 is difficult because when I connect espbell in the correct way it opens forever, even without power
2 testing
3 yes it does

**Reply by PilaScat** [reply]
Ok maybe the relay is damaged, even with bell cable disconnected and no power to the espbell it opens

**Reply by PricelessToolkit** [reply]
> Ok maybe the relay is damaged, even with bell cable disconnected and no power to the espbell it opens

Yep. the gate opener is 24v not 220v?

**Reply by PricelessToolkit** [reply]
Please send me a message taught the shop contact form.

**Reply by PilaScat** [reply]
> > Ok maybe the relay is damaged, even with bell cable disconnected and no power to the espbell it opens
> 
> Yep. the gate opener is 24v not 220v?

it should
![352661984-b8b6462b-a9f7-41c2-85d2-17718e836e4d](https://github.com/user-attachments/assets/9ab5fa68-971d-43e1-8b23-5761737a65cf)
is this?

**Reply by PricelessToolkit** [reply]
it is ok only 20VA. PS;  I send you a message.

**Reply by PilaScat** [reply]
> it is ok only 20VA. PS; I send you a message.

I saw, thank you 🫶

**Comment by PilaScat** [comment]
Installed, the first time it opened correctly, the second time same problem as before, relay damaged, correctly wired, I don't know why
Is very sad, I wanted it so much 😭

**Reply by PilaScat** [reply]
```
[11:56:39][I][safe_mode:041]: Boot seems successful; resetting boot loop counter
[11:57:26][D][sensor:093]: 'ESPBell-LITE WiFi Signal': Sending state -64.00000 dBm with 0 decimals of accuracy
[11:57:35][D][sensor:093]: 'ESPBell-LITE Uptime': Sending state 120.20000 s with 0 decimals of accuracy
[11:58:26][D][sensor:093]: 'ESPBell-LITE WiFi Signal': Sending state -63.00000 dBm with 0 decimals of accuracy
[11:58:35][D][sensor:093]: 'ESPBell-LITE Uptime': Sending state 180.20000 s with 0 decimals of accuracy
[11:59:07][D][binary_sensor:036]: 'Doorbell': Sending state ON
[11:59:07][D][binary_sensor:036]: 'Doorbell': Sending state OFF
[11:59:17][D][switch:012]: 'Gate' Turning ON.
[11:59:17][D][switch:055]: 'Gate': Sending state ON
[11:59:17][D][switch:016]: 'Gate' Turning OFF.
[11:59:17][D][switch:055]: 'Gate': Sending state OFF
[11:59:25][D][switch:012]: 'Gate' Turning ON.
[11:59:25][D][switch:055]: 'Gate': Sending state ON
[11:59:25][D][switch:016]: 'Gate' Turning OFF.
[11:59:25][D][switch:055]: 'Gate': Sending state OFF
[11:59:26][D][sensor:093]: 'ESPBell-LITE WiFi Signal': Sending state -64.00000 dBm with 0 decimals of accuracy
[11:59:35][D][sensor:093]: 'ESPBell-LITE Uptime': Sending state 240.20399 s with 0 decimals of accuracy
WARNING espbell-lite @ 192.168.1.100: Connection error occurred: Ping response not received after 90.0 seconds
```
last logs before disconnecting power, I clicked the switch two times, the first opened correctly, the second started the loop
(I changed the SSR ON time to 0.5s because it was sufficient)
[espbell-lite.yaml.txt](https://github.com/user-attachments/files/16824276/espbell-lite.yaml.txt)
my config

**Reply by PricelessToolkit** [reply]
Your solenoid which pushes the lock uses much more than 2.4A or is powered with more than 24V. I think for that reason wires for "7" terminals are thick.

**Reply by PilaScat** [reply]
You saw the power supply yourself, the only thing is that connected to the button there are also two relay buttons to open it from other places in the house, but I don't think that's the problem
The cables are thick because they were the only ones that were in the house

**Reply by PricelessToolkit** [reply]
It seems to me that there is a second power supply or the solenoid uses a direct connection to 220v. You can check it with a multimeter.

**Reply by PilaScat** [reply]
It would be very dangerous, I have an iron gate ahaha
I'll try to investigate with the multimeter

**Reply by PilaScat** [reply]
Ok 17v pass in the switch, 12v are used when opening, ampere I can't read them I don't know why, but it seems less the 2A

**Reply by PilaScat** [reply]
1.6A

**Reply by PricelessToolkit** [reply]
The SSR can handle more than 1.6A unless there is a surge of more than 5A and 100ms.

From datasheet
![ssr](https://github.com/user-attachments/assets/c15d9d5d-4b17-4089-b376-788757e4a6e9)

I have no more ideas about how to diagnose the cause of the two-time failure of the SSR.

**Reply by PilaScat** [reply]
I very don't know why 😭
This week I'll call a friend who knows a little more than me and let's try to understand what the cause could be.
I'll keep you updated

**Comment by PilaScat** [comment]
Here I am again, I'm trying to fix it together with a friend who knows more than me, do you know where I can buy the same Solid State Relay or maybe even more powerful?

**Reply by PricelessToolkit** [reply]
https://www.lcsc.com/product-detail/Solid-State-Relays-MOS-Output-PhotoMOS_Letex-LT218_C2691607.html i don't know more powerful in the same form factor. try with any generic  3.3v SSR from ali.

**Comment by PilaScat** [comment]
I'm bringing this thread back to add something. I tried again, and we discovered that my system is carrying a quantity of amperes that's difficult to measure even with a multimeter, and obviously much higher than the relay can handle. We blew two 10A fuses, but at least by replacing the ESPBELL relay, everything is back to working order.
I'm not sure how to proceed now, given the load. I was thinking of keeping the ESPBELL only for the Ding and using a Sonoff for the lock.
Any other ideas would be appreciated.
Thanks.

**Reply by PricelessToolkit** [reply]
10A is not a jock. You can enable another powerful relay with espbell relay.

**Reply by PilaScat** [reply]
Could you iterate? Some suggestion?
Easier than a separate sonoff?

**Reply by PricelessToolkit** [reply]
You can use the ESPBELL relay to switch on and off a more powerful relay, (coil-driven). That way, the ESPBELL isn’t directly handling the high current just triggering the bigger relay. It’s not necessarily easier than using a separate Sonoff, but you’ll keep all the control logic in the same device

**Comment by PilaScat** [comment, marked as answer]
![20250725_105323](https://github.com/user-attachments/assets/6dc3dba1-44e2-4ad0-abe6-776b9da1bec6)
I finally did it!
It works! 🎉🎉🥳

**Reply by PricelessToolkit** [reply]
I'm glad everything worked out!!!!!!!!!!!
